less/home-page.less

```diff
@@ -17,15 +17,33 @@
       width: 48%;
       padding-left: 1%;
 
+      @media @mobile {
+        margin-left: -100px;
+        display: flex; 
+        flex-direction: column;
+        align-items: center; 
+      }
+
       img {
         border-radius: 10px;
+        
+        @media @mobile {
+          margin-top: 189px;
+          margin-left: -250px;
+           
+        }
       }
     }
   }
 
   .inverse-content {
     padding-bottom: 30px;
-    border-bottom: 2px dashed @navigation-border;
+    border-bottom: 2px dashed @navigation-border; 
+     
+    @media @mobile {
+      border-bottom: 2px dashed @navigation-border;
```

Comment:
Using `@navigation-border` here can be a bit misleading since what you're actually setting here is the colour. Since it is shared between all borders on the page, you could simply rename it to `@border-color` which would make more sense to other developers who read it.